package.json

```diff
@@ -101,6 +101,7 @@
 		"uglify-js": "^3.4.9"
 	},
 	"devDependencies": {
+		"mkdirp": "^0.5.1",
```

Comment:
should this be under depenedencies? since we call `build:sass` and `build:js` when deploying right?

Reply:
Yeah we should move it, good point